torch/csrc/distributed/rpc/python_functions.cpp

```diff
@@ -128,8 +153,14 @@ std::shared_ptr<FutureMessage> pyRpcBuiltin(
   py::gil_scoped_release release;
   auto scriptCall = std::make_unique<ScriptCall>(op, std::move(stack));
   auto agent = RpcAgent::getCurrentRpcAgent();
-  return sendMessageWithAutograd(
+  auto responseMessageFuture = sendMessageWithAutograd(
       *agent, dst, std::move(*scriptCall).toMessage(), false, rf);
+
+  // Notice, even we can get the JIT type of the Python object from the op
+  // schema, there is no need to do that, because the return value will be
+  // passed back to Python land eventually.
+  return std::make_shared<torch::jit::PythonFutureWrapper>(
+      wrapFutureMessageInJitFuture(responseMessageFuture));
```

Comment:
Why we don't need the same `handleException` as in `pyRpcPythonUdf`? Builtin operator execution can also hit errors, no?

Reply:
This is only for unwrap RemoteException, which is only added for pythoon_udf rpc code path.

https://github.com/pytorch/pytorch/blob/5d92a6cc30013a3687d4f82ad5030bd047ed9755/torch/csrc/distributed/rpc/python_rpc_handler.cpp#L107